app.py

```diff
@@ -595,6 +608,126 @@ def call_api(url, params=None):
     return data
 
 
+# get each team's rostered players
+def get_each_teams_players():
+    league_id = request.args.get('leagueId')
+    url = f'https://fantasy.espn.com/apis/v3/games/fba/seasons/{app.config.get("SEASON")}/segments/0/leagues/{league_id}'
+    params = {
+        'view': 'mRoster'
+    }
+    data = call_api(url, params)
+    team_rosters = {}
```

Comment:
This object holds so many dictionaries 🤯🤯🤯. It's not a bad thing, just a general comment. Most of my experience is in Java where everything is strongly typed.

Reply:
Yeah this thing is MASSIVE